python/mxnet/symbol/numpy/_symbol.py

```diff
@@ -31,13 +31,95 @@
 
 __all__ = ['zeros', 'ones', 'maximum', 'minimum', 'stack', 'concatenate', 'arange', 'argmax',
            'clip', 'add', 'subtract', 'multiply', 'divide', 'mod', 'power', 'split', 'swapaxes',
-           'expand_dims', 'tile', 'linspace', 'sin', 'cos', 'sinh', 'cosh', 'log10', 'sqrt']
+           'expand_dims', 'tile', 'linspace', 'sin', 'cos', 'sinh', 'cosh', 'log10', 'sqrt', 
+           'absolute', 'cbrt', 'arccos']
 
+@set_module('mxnet.symbol.numpy')
```

Comment:
There unary ops' location may need to be at the bottom of this file

Reply:
Thx. It also works.